File: dataset/java2python/java/SHUFFLE_A_GIVEN_ARRAY.java
int[] f_gold ( int arr [ ] , int n ) {
  Random r = new Random ( ) ;
  for ( int i = n - 1 ; i > 0 ; i -- ) {
    int j = r . nextInt ( i + 1 ) ;
    int temp = arr [ i ] ;
    arr [ i ] = arr [ j ] ;
    arr [ j ] = temp ;
  }
  return arr ;
}

File: result/translate/transcoder/java2python/python/SHUFFLE_A_GIVEN_ARRAY.py
def f_gold ( arr , n ) :
    r = random.Random ( )
    for i in range ( n - 1 , 0 , - 1 ) :
        j = r.randint ( i + 1 , i + 1 )
        temp = arr [ i ]
        arr [ i ] = arr [ j ]
        arr [ j ] = temp
    return arr